src/main/java/com/stablekernel/standardlib/LoadingImageView.java

```diff
@@ -0,0 +1,70 @@
+package com.stablekernel.standardlib;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.AttrRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+import android.util.TypedValue;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.ProgressBar;
+
+import static android.view.ViewGroup.LayoutParams.WRAP_CONTENT;
+
+public class LoadingImageView extends FrameLayout {
+    private ImageView imageView;
+    private ProgressBar progressBar;
+
+    public LoadingImageView(@NonNull Context context) {
+        this(context, null);
+    }
+
+    public LoadingImageView(@NonNull Context context, @Nullable AttributeSet attrs) {
+        this(context, attrs, 0);
+    }
+
+    public LoadingImageView(@NonNull Context context, @Nullable AttributeSet attrs, @AttrRes int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+
+        View view = LayoutInflater.from(context).inflate(R.layout.loading_image_view, this, true);
+        imageView = (ImageView) view.findViewById(R.id.imageView);
+        progressBar = (ProgressBar) view.findViewById(R.id.progressBar);
+
+        final TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.LoadingImageView);
+
+        Drawable image = a.getDrawable(R.styleable.LoadingImageView_imageSrc);
+        int imageHeight = a.getDimensionPixelSize(R.styleable.LoadingImageView_imageHeight, dipsToPix(WRAP_CONTENT));
+        int imageWidth = a.getDimensionPixelSize(R.styleable.LoadingImageView_imageWidth, dipsToPix(WRAP_CONTENT));
```

Comment:
1. If you use `https://developer.android.com/reference/android/content/res/TypedArray.html#getDimension(int, float)` instead of `getDimensionPixelSize` you don't need to do any dipsToPix conversion.... 

2. I'm not sure `dipsToPix(WRAP_CONTENT)` is working like you think. (I'd need to see an implementation to be sure).  `WRAP_CONTENT` is just an int: `public static final int WRAP_CONTENT = -2;`. So you are trying to convert `-2` to dp and then setting that as image height/width.